Trie/PrefixTree.rb

```diff
@@ -0,0 +1,112 @@
+require_relative "PrefixTreeNode"
```

Comment:
PrefixTree.rb is camelcase, but we use snakecase in ruby

Reply:
fixed